src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/PropertyAccessor.cs

```diff
@@ -63,12 +103,12 @@ protected PropertyAccessor(Type type, Dictionary<string, CompiledIndexField> gro
                 {
                     foreach (var groupByField in groupByFields.Values)
                     {
-                        if (groupByField.IsMatch(prop.Name))
-                        {
-                            getMethod.GroupByField = groupByField;
-                            getMethod.IsGroupByField = true;
-                            break;
-                        }
+                        if (!groupByField.IsMatch(prop.Name)) 
```

Comment:
Can you change the negation into `== false`? We use it everywhere.

Reply:
Done